taccsite_cms/static/site_cms/css/src/_imports/tools/x-drop-cap.css

```diff
@@ -1,19 +1,12 @@
 .x-drop-cap {
-  /* Drop the letter */
-  float: left;
   /* Estimate horizontal spacing to match magazine design */
   /* WARN: Firefox calculates `margin-…` differently here, so use `padding-…` */
   padding-right: 0.125em; /* approx: `1em` of normal text */
 
   font-size: 6em; /* approx: 3 lines, of normal text, tall */
```

Comment:
**Opportunity for less code.** Having font-size seems unnecessary. I tried without it and `initial-letter` resized the font for me. I did need to change the padding-right, though, because `em` became based off of parent's font-size.